bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/discovery/HueBridgeDiscoveryParticipant.java

```diff
@@ -44,6 +52,15 @@
 @Component(service = UpnpDiscoveryParticipant.class)
 public class HueBridgeDiscoveryParticipant implements UpnpDiscoveryParticipant {
 
+    private final Logger logger = LoggerFactory.getLogger(HueBridgeDiscoveryParticipant.class);
+
+    // Hue bridges have maxAge 100 seconds, so set the default grace period to half of that
+    private long removalGracePeriodSeconds = 50;
+
+    @Reference
+    @Nullable
+    ConfigurationAdmin configAdmin;
```

Comment:
Why not have this supplied in the constructor so you can make this non-null and final?

Reply:
Good idea. Done.